Chef Server -> Chef Infra Server (#2132)
chef/chef-web-docs: chef_master/source/push_jobs.rst

=====================================================
Chef Push Jobs has the following requirements:

> * An on-premises Chef Server; Chef Push Jobs is not supported when running Chef with the hosted Chef server

Reviewer: ```suggestion
* An on-premises Chef Infra Server. Hosted Chef does not support Chef Push Jobs.
```

> * To use the **push-jobs** cookbook to configure the Chef Push Jobs client, Chef Infra Client must also be present on the node (because only Chef Infra Client can use a cookbook to configure a node)

Reviewer: ```suggestion
* The Chef Push Jobs client can be configured using a **push-jobs** cookbook, but Chef Infra Client must also be present on the node. Only Chef Infra Client can use a cookbook to configure a node.
```
